md.obsidian.Obsidian.yml

```diff
@@ -0,0 +1,54 @@
+app-id: md.obsidian.Obsidian
+branch: master
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: obsidian.sh
+tags:
+  - proprietary
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+modules:
+  - name: obsidian
+    buildsystem: simple
+    build-commands:
+      - mkdir -p bin share/icons/hicolor/{16x16,32x32,48x48,64x64,128x128,256x256,512x512}/apps share/applications
+      - mv md.obsidian.Obsidian-16.png  share/icons/hicolor/16x16/apps/md.obsidian.Obsidian.png
```

Comment:
An 512x512 should be enough, it will be downscaled by appstream-util to smaller size for "internal" use.